test/cmd/legacy-script.sh

```diff
@@ -885,6 +885,11 @@ runTests() {
     record_command run_kubectl_explain_tests
   fi
 
+  ###########################
+  # CRD Deletion / Re-creation #
+  ###########################
```

Comment:
Nit (sorry), but since all other other ones line up the boxes, might as well here too I guess...
```suggestion
  ##############################
  # CRD Deletion / Re-creation #
  ##############################
```